### ISSUE TITLE
GitHub App integration to Slack does not work with this repository

### ISSUE THREAD

**Issue by tohov** [Azure/terraform-azurerm-caf-enterprise-scale#166]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

**Cannot establish the integration to Slack using GitHub App for Slack**

Somehow this repository cannot be found using the GitHub App for Slack. I'd like to subscribe to the new releases using the GitHub App which would post a message to the slack channel when new release is published. 

GitHub App uses the following syntax: _/github subscribe microsoft/azure-pipelines-tasks releases_
That command would work, but running the same command using this repository will fail. One can also paste the complete URL to the repo after keyword subscribe.

Could you please check if there is a setting in your repository that is disallowing (I assume this is using some kind of webhook) this operation? It would be nice to automatically get notified of new releases or issues. I understand this is not an issue in this module or code, but something that is nice to have and most likely just one setting that is preventing fans of this repo of using the GitHub App for Slack.

**Steps to Reproduce**

1. Install GitHub App on your Slack organization (https://github.com/integrations/slack#configuration)
2. Run _/github subscribe microsoft/azure-pipelines-tasks_ to subscribe to the repo that works
3. Run _/github subscribe azure/terraform-azurerm-caf-enterprise-scale_ to subscribe to this repo. Does not work.
4. Run _/github unsubscribe microsoft/azure-pipelines-tasks_ to unsubscribe to the repo that worked before.

**Screenshots**

I assure that we have the GitHub App installed since it is working fine with other repositories.
![image](https://user-images.githubusercontent.com/38506856/133388657-40a04375-09d2-4764-8f1b-4a91fb5a24a6.png)

**Comment by jtracey93** [comment]
Hi @tohov, Are you able to integrate with any repo in the Azure GitHub org out of interest? Say the main Enterprise-scale one? Azure/Enterprise-scale

**Comment by tohov** [comment]
No, actually I'm not. I tried Azure/Enterprise-scale and Azure/azure-quickstart-templates. Seems like there are different settings in Microsoft org and Azure org. Is there anyone you could escalate this issue to?

**Comment by jtracey93** [comment]
Hi @tohov, 

Thanks for the quick response. So to confirm, you **can** for the the `Microsoft` org but **cannot** for the `Azure` org?

I have already started a discussion via our channels internally, but the above information would be really useful 👍

Let me know

Thanks

Jack

**Comment by tohov** [comment]
Yes, I confirm. I **can** use the feature for Microsoft org but **cannot** for the Azure org.

**Comment by tohov** [comment]
Something in this org and repository has been done right to allow the app to use webhook to post messages into Slack: https://github.com/microsoft/azure-pipelines-tasks

**Comment by jtracey93** [comment]
Cool thanks for the update. I have passed that on internally. Will keep you posted.

Thanks

**Comment by jtracey93** [comment]
Hey @tohov,

Could you try the integration again to this repo? Not saying it will work but may have enabled it based on internal feedback?

Let me know

Thanks

**Comment by tohov** [comment]
@jtracey93 It works now. Nice! Really awesome! Thank you so much. Following new releases just got so much easier for us.
